core/src/main/scala/akka/persistence/r2dbc/internal/R2dbcExecutor.scala

```diff
@@ -50,7 +50,7 @@ import org.slf4j.Logger
 
         override def onComplete(): Unit = {
           subscription = null
-          if (promise.isCompleted)
+          if (promise.isCompleted) // I guess what we need here is: if not (yet) completed, we complete with failure
```

Comment:
ok, ignore. This has been already spotted in https://github.com/akka/akka-persistence-r2dbc/pull/40/files#r731926737